tcms/testplans/templates/testplans/mutable.html

```diff
@@ -7,13 +7,21 @@
 {% endblock %}
 
 <head>
-    <title>{% block title %}{% trans "Create new TestPlan" %}{% endblock %}</title>
+    <title>
+        {% block title %}
+            {% if test_plan %}
+                {% trans "Edit TestPlan" %}
+            {% else %}
+                {% trans "Create new TestPlan" %}
+            {% endif %}
+        {% endblock %}
+    </title>
 </head>
 
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-        <form class="form-horizontal" action="{% url 'plans-new' %}" method="post">
+        <form class="form-horizontal" action="{% if test_plan %} {% url "plan-edit" test_plan.plan_id %} {% else %}{% url 'plans-new' %} {% endif %}" method="post">
```

Comment:
Use single quotes for URL value so you are consistent with the rest of the line and also because we have double quotes surrounding the entire expression.